pytorch_lightning/strategies/launchers/base.py

```diff
@@ -34,3 +34,6 @@ def is_interactive_compatible(self) -> bool:
     @abstractmethod
     def launch(self, function: Callable, *args: Any, **kwargs: Any) -> Any:
         """Launches the processes."""
+
+    def teardown(self) -> None:
+        """This method is called to teardown the launcher."""
```

Comment:
we need to be carful. how do we precisely define a teardown for a launcher? 
If you look at where the process boundaries are when comparing ddp vs ddp spawn, it's vastly different. 

Can we just remove it from the base class for now and call it at the end of the launch() method internally here on the specific implementation?